src/main.rs

```diff
@@ -217,37 +217,40 @@ Global config");
             }
         };
 
-        cb.signature(signature.to_owned());
+        config_builder.signature(signature.to_owned());
     }
 
     // In case we are in write-mode AND release mode,
     // we will make sure we got all configuration settings
     if !is_dry_run && release_mode {
-        let remote_url = match repo.find_remote("origin") {
-            Err(e) => print_exit!("Could not determine the origin remote url: {:?}", e),
-            Ok(remote) => {
-                let url = remote.url().expect("Remote URL is not valid UTF-8");
-                Url::parse(&url).expect("Remote URL can't be parsed")
-            }
-        };
-
-        let (user, repo_name) = user_repo_from_url(remote_url)
-            .unwrap_or_else(|e| print_exit!("Could not extract user and repository name from URL: {:?}", e));
-        cb.user(user);
-        cb.repository_name(repo_name);
-
-        let gh_token = env::var("GH_TOKEN")
-            .unwrap_or_else(|err| print_exit!("GH_TOKEN not set: {:?}", err));
-
-        let cargo_token = env::var("CARGO_TOKEN")
-            .unwrap_or_else(|err| print_exit!("CARGO_TOKEN not set: {:?}", err));
-
-        cb.gh_token(gh_token);
-        cb.cargo_token(cargo_token);
+        let remote_or_none = repo.find_remote("origin");
+        if remote_or_none.is_err() {
+            let err = remote_or_none.err().unwrap();
+            logger::warn(format!("Could not determine the origin remote url: {:?}", err));
+            logger::warn("semantic-rs therefor does not push anything to a remote repository does not create a release on GitHub");
```

Comment:
Afaik it should be `therefore` (yey, english).

What about a rewording?
"semantic-rs can't push changes or create a release on GitHub"